src/createRollupPrepareTransactionRequest.unit.test.ts

```diff
@@ -172,3 +168,68 @@ it(`fails to prepare transaction request if "params.nativeToken" doesn't use 18
     `"params.nativeToken" can only be configured with a token that uses 18 decimals.`,
   );
 });
+
+it(`successfully prepares a transaction request with the default rollup creator`, async () => {
```

Comment:
```suggestion
it(`successfully prepares a transaction request with the default rollup creator and a gas limit override`, async () => {
```
nitpick, same for the test below (feel free to disregard)